src/layouts/ExampleLayout.astro

```diff
@@ -19,3 +19,9 @@ const { title } = Astro.props;
     <slot />
   </body>
 </html>
+
+<style>
+  body {
+    padding: var(--size-spacing-md);
```

Comment:
The example component was stuck to the edge and difficult to see, so padding was added.